packages/sit-onyx/src/i18n/locales/de-DE.json

```
@@ -84,5 +86,21 @@
   "showTooltip": {
     "info": "Info-Tooltip anzeigen",
     "error": "Fehler-Tooltip anzeigen"
+  },
+  "colorScheme": {
+    "headline": "Erscheinungsbild ändern",
+    "subtitle": "Bitte wählen das Erscheinungsbild der Anwendung",
+    "auto": {
+      "label": "Auto",
+      "description": "Die Anwendung passt ihr Aussehen automatisch an deine Systemeinstellungen an."
+    },
+    "light": {
+      "label": "Hell",
+      "description": "Der Modus \"Hell\" bietet einen optimalen Kontrast für helle Umgebungen."
```

```suggestion
      "description": "Der helle Modus bietet einen optimalen Kontrast für helle Umgebungen."
```

in english we also don't have extra "", so this would be closer to the OG text?